core/src/main/java/com/yahoo/tagchowder/CommandLine.java

```diff
@@ -153,7 +153,7 @@ private static void process(final String src, final OutputStream os) throws IOEx
         } else {
             r = new Parser();
         }
-        theSchema = new HTMLSchema(true);
+        theSchema = new HTMLSchema(new ParserContext(true));
```

Comment:
get the Context from the parser instance above.

Reply:
done